sycl/plugins/esimd_cpu/pi_esimd_cpu.cpp

```diff
@@ -652,6 +416,9 @@ sycl::detail::ESIMDDeviceInterface::ESIMDDeviceInterface() {
   sycl_get_surface_base_addr_ptr = cm_support::get_surface_base_addr;
   __cm_emu_get_slm_ptr = cm_support::get_slm_base;
   cm_slm_init_ptr = cm_support::init_slm;
+
+  sycl_get_cm_buffer_params_ptr = sycl_get_cm_buffer_params;
```

Comment:
I don't see where sycl_get_cm_buffer_params_ptr is defined.

Reply:
It is defined in another PR - https://github.com/intel/llvm/pull/4020.

I'll add same definition here and resolve conflict later if any.

Reply:
Definition is added in this PR.